client/src/components/Grid/configs/visualizations.js

```diff
@@ -0,0 +1,192 @@
+import axios from "axios";
+
+import Filtering, { contains, equals, expandNameTagWithQuotes, toBool } from "@/utils/filtering";
+import { withPrefix } from "@/utils/redirect";
+import { errorMessageAsString, rethrowSimple } from "@/utils/simple-error";
+
+/**
+ * Request and return data from server
+ */
+async function getData(currentPage, perPage, sortBy, sortDesc, searchTerm) {
+    const offset = perPage * (currentPage - 1);
+    let q = `/api/visualizations/detailed?limit=${perPage}&offset=${offset}&sort_by=${sortBy}&sort_desc=${sortDesc}`;
+    if (searchTerm) {
+        q += `&search=${searchTerm}`;
+    }
+    const response = await axios.get(withPrefix(q));
```

Comment:
It would be great now that you have migrated those routes to FastAPI to convert this module to Typescript and use the `fetcher` instead of `axios` to take full advantage of it :)